connect/connect-core/connect-element-templates/element-template-api/src/test/java/io/miragon/miranum/connect/elementtemplate/impl/ElementTemplateInfoMapperTest.java

```diff
@@ -60,12 +60,12 @@ private Method getMethod(String methodName, Class<?>... parameterTypes) {
         }
     }
 
-    @GenerateElementTemplate(name = "name", id = "id", version = 0.1, type = "type")
+    @GenerateElementTemplate(name = "name", id = "id", version = 01, type = "type")
```

Comment:
01 gets evaluated to 1 if it is an integer, no point of writing it like that I guess. Maybe we can let it be a double and just adjust the filename creation. Is still useful to have a version of 1.2 for example

Reply:
But how would you want to name a file if there is a version 1.2? 
Just without a dot? This might be conflicting with the version 12 then...

Reply:
String(1.2).split("//.") I guess but looks too much like String magic to me. But after some research stackoverflow seems to agree on double as a bad idea. The suggestions are to use a string or to separate the major and minor version into two integers. We might just create a simple class to make things more clear here. What do you think?

Reply:
Thanks for doing some research on this @rami-nk 💯 
What about using something `-` as separator between minor and major versions?